### ISSUE TITLE
Info window disappears on cluster markers when zooming in/out.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Cemberk/android-maps-extensions#9]
```
What steps will reproduce the problem? Please provide sscce (short, self
contained, correct example) when possible.

1) Run demo app
3) Click Narvik / Tromsr (Norway) cluster
4) Zoom in

What is the expected output? What do you see instead?

Info window disappears, when cluster is not changed in any way (markers count 
inside).
```

Original issue reported on code.google.com by `maciek.g...@gmail.com` on 14 Apr 2013 at 11:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `maciek.g...@gmail.com` on 14 Apr 2013 at 11:42

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3
```

Original comment by `maciek.g...@gmail.com` on 8 May 2013 at 7:15
- Changed state: **Fixed**
